requirements.txt

```diff
@@ -4,16 +4,15 @@
 #
 #    pip-compile --output-file requirements.txt requirements.in
 #
-
 backports.ssl-match-hostname==3.5.0.1  # via docker-py
 click==6.6
 docker-py==1.10.6
 docker-pycreds==0.2.1     # via docker-py
-hubstorage==0.23.1
```

Comment:
I did, it depends on which python you use to compile the dependencies list. On python 3 it drops `ipaddress` and SSL backports packages as you say, but for python 2 it keeps it as-is.